src/Address.svelte

```diff
@@ -2,29 +2,36 @@
   import { onMount } from 'svelte';
   import { link } from 'svelte-routing';
   import { ethers } from 'ethers';
-  import { explorerLink, safeLink } from '@app/utils';
-  import Blockies from '@app/Blockies.svelte';
+  import { safeLink } from '@app/utils';
   import Loading from '@app/Loading.svelte';
   import type { Config } from '@app/config';
+  import type { Registration } from '@app/base/registrations/registrar';
+  import Avatar from "@app/Avatar.svelte";
+  import { getRegistration } from '@app/base/registrations/registrar';
```

Comment:
Maybe group the imports that come from the same file?

Reply:
The issue is that we cannot group a type import with a function import.

Reply:
Oh I just meant having the import lines next to each other :grimacing: